README.md

```diff
@@ -41,6 +41,16 @@ Other than that, as `sbt-dynver` is an AutoPlugin that is all that is required.
 | when there are no commits, or the project isn't a git repo           | HEAD+20140707-1030             | true       |
 ```
 
+## Tag Requirements
+
+In order to be recognized by sbt-dynver, tags must meet the following two requirements:
+1. Must be an annotated tag (as opposed to a lightweight tag)
```

Comment:
The requirement that tags must be annotated/non-lightweight was actually lifted in [v1.1.0](https://github.com/dwijnand/sbt-dynver/releases/tag/v1.1.0), see #15. So can I ask you to remove this line please?